streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -300,23 +308,25 @@ public void checkpoint(final Map<TopicPartition, Long> ackedOffsets) {
         // write the checkpoint file before closing, to indicate clean shutdown
         try {
             checkpoint.write(checkpointedOffsets);
-        } catch (IOException e) {
+        } catch (final IOException e) {
             log.warn("Failed to write checkpoint file to {}", new File(baseDir, CHECKPOINT_FILE_NAME), e);
         }
     }
 
-    private int getPartition(String topic) {
-        TopicPartition partition = partitionForTopic.get(topic);
+    private int getPartition(final String topic) {
+        final TopicPartition partition = partitionForTopic.get(topic);
 
         return partition == null ? taskId.partition : partition.partition();
     }
 
     void registerGlobalStateStores(final List<StateStore> stateStores) {
-        for (StateStore stateStore : stateStores) {
+        log.info("{} Register global stores", logPrefix);
```

Comment:
I had the same though be did not to it, because it will be the same log message for all processors... quite repetitive. Can add it though.